cmd/ack-generate/command/release.go

```diff
@@ -61,6 +61,10 @@ func generateRelease(cmd *cobra.Command, args []string) error {
 	if optReleaseOutputPath == "" {
 		optReleaseOutputPath = filepath.Join(optServicesDir, svcAlias)
 	}
+	if optImageRepository == "" {
+		// TODO(vijat): Replace u2r4f3v7 with aws-controllers-k8s
+		optImageRepository = fmt.Sprintf("public.ecr.aws/u2r4f3v7/%s-controller", svcAlias)
```

Comment:
We override with `--image-repository` in the script anyway. I don't think it's necessary to use the temporary URL in this one.

Reply:
Yeah, agreed. If we can override the `u2r4f3v7` thing for a short while in an env var, I think we're good.